docs/user-guide/import-certificates.md

```diff
@@ -1,13 +1,172 @@
-# Configure certificates if you have existing certificates
+# Import and configure an existing certificate 
 
-## Import your PKCS12 certificate
+Zowe makes it possible to import and configure existing certificates. The following certificates are supported by Zowe:
+* PKCS12 certificates
+* JCERACFKS certificates 
+
+Review the procedure to import existing PKCS12 or JCERACFKS certificates as well as how to import a certificate stored in an MVS data set into a Zowe keyring.
+## Import an existing PKCS12 certificate
+
+To import a PKCS12 certificate, it is first necessary to import a certificate authority. Then, you can configure the section `>>>> Certificate setup scenario 2` in `zowe.yaml`. 
```

Comment:
Are there links to these certificate setup scenarios?

Reply:
> Are there links to these certificate setup scenarios?
Done.